lib/common/ipc_client.c

```diff
@@ -104,8 +99,7 @@ pcmk_new_ipc_api(pcmk_ipc_api_t **api, enum pcmk_ipc_server server)
         return ENOMEM;
     }
 
-    (*api)->ipc = crm_ipc_new(pcmk_ipc_name(*api, false),
-                              (*api)->ipc_size_max);
```

Comment:
This would still be doing something if we used the maximum of `max_size` and 128k to preserve behavior. As mentioned in another comment, this removes functionality.

As long as, like you said, these commits don't get merged until the whole "split messages" scheme is finished, I suppose that's fine.

Reply:
If we ARE going to make `max_size` do nothing in `crm_ipc_new()`, then it's probably clearer to have all the callers pass 0 *first* really (deprecating whatever and making whatever API commits are necessary). So by the time we get to ignoring `max_size`, it doesn't change behavior anymore.